bibtexparser/bibtexexpression.py

```diff
@@ -149,9 +149,22 @@ def __init__(self):
         entry_type.setParseAction(first_token)
 
         # Entry key: any character up to a ',' without leading and trailing
-        # spaces.
-        key = pp.SkipTo(',')('Key')  # Exclude @',\#}{~%
-        key.setParseAction(lambda s, l, t: first_token(s, l, t).strip())
+        # spaces. Also exclude spaces and prevent it from being empty.
+        key = pp.SkipTo(',')('Key')  # TODO Maybe alsi exclude @',\#}{~%
+
+        def citekeyParseAction(s, l, t):
```

Comment:
Please, add a short docstring to specify what do s, l and t correspond to. I'm not sure it's straightforward for everyone, even for the comment above.

Reply:
`s, l, t` is the standard signature of `pyparsing` parse actions. It is indeed not clear so I will change to `string_, location, token`.